graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsConfigurationProperties.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.autoconfig
```

Comment:
Would it be useful to do #213 after this one? Or that I include the name change for OSS already in this PR?
I would like to avoid conflicts.

Reply:
I thought that as well as It would minimize conflict and easy to knock out here in this MR. However, that auto-config is "open" so the rename will be considered a breaking change and I figured we did not want to have to juggle that in this proposal as well.

Reply:
The only reason the auto config classes are "open" is because it's required by Spring (for proxying I think). They are not considered API classes though, so changing a name isn't really breaking.